app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -291,6 +291,36 @@ class NavigationOverlayFragment : Fragment() {
             }
     }
 
+    private fun createChannelFactory(): DefaultChannelFactory = DefaultChannelFactory(
+            loadUrl = { urlStr ->
+                if (urlStr.isNotEmpty()) {
+                    onNavigationEvent.invoke(NavigationEvent.LOAD_TILE, urlStr, null)
+                }
+            },
+            onTileLongClick = openHomeTileContextMenu,
+            onTileFocused = {
```

Comment:
Not sure if this should be in the Factory, if PinnedTiles are the only ones that can be unpinned.

Reply:
Same with long click.

Reply:
Though maybe that's fine, if it's already here. Who knows what might change with our future tiles (like dismissing Pocket tiles, the way Desktop does it).

Reply:
I'm unsure on this, tbh.  I timeboxed trying to find a good removal solution, and didn't manage to.  I have that as part of the follow up.